lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -27,15 +27,9 @@ def initialize(build_mechanism,
       @skip_ci_status = skip_ci_status
     end
 
-    def deploy_cli_hook(parser)
-      parser.on('-s', '--skip-ci-status', 'Skips checks on CI jobs', FalseClass) do |value|
-        @skip_ci_status = value
-      end
-    end
-
     def build_cli_hook(parser)
-      parser.on('-s', '--skip-ci-status', 'Skips checks on CI jobs', FalseClass) do |value|
-        @skip_ci_status = value
+      parser.on('-s', '--skip-ci-status', 'Skips checks on CI jobs') do |value|
+        @skip_ci_status = true if value
```

Comment:
If I understand correctly, @askreet suggested this:
`parser.on('-s', '--skip-ci-status', 'Skips checks on CI jobs', FalseClass)`

I suppose the last parameter defines a default value, therefore it can be simplified to something like this:
`@skip_ci_status = !!value`
or even perhaps just
`@skip_ci_status = value`

Same applies for the other hook defined in the previous PR.
@askreet can you verify this please?

Reply:
No need for default value or even assign the received value.. because of no value sent in command parameter .. so if this flag sent that means true else there is default value in constructor